reports/serializers.py

```
@@ -1,8 +1,51 @@
+from typing import List
+
 from reports.models import EnrolmentReport
 from rest_framework import serializers
 
 
+class Size2ArrayFieldBase(serializers.Field):
+    def __init__(self, first_value_name: str = "id", second_value_name: str = "label"):
+        self.first_value_name = first_value_name
+        self.second_value_name = second_value_name
+        super().__init__()
+
+    def _array_to_dict(self, entry: List[str]):
+        if len(entry) == 2:
+            return {self.first_value_name: entry[0], self.second_value_name: entry[1]}
+        return None
```

Do we really want to just silently ignore this error situation here? 🙂